www/src/components/showcase-details.js

```diff
@@ -252,25 +258,6 @@ const ShowcaseDetails = ({ parent, data, isModal, categories }) => (
             >
               <Helmet>
                 <title>{data.sitesYaml.title}: Showcase | GatsbyJS</title>
-                <meta
-                  property="og:image"
-                  content={`https://www.gatsbyjs.org${
```

Comment:
In my comment below, you can see how this would be useful, e.g. no need to prefix with this hard coded string.